package.json

```diff
@@ -1,8 +1,24 @@
 {
+	"private": true,
+	"name": "botbuilder-packages",
+	"scripts": {
+		"postinstall": "lerna bootstrap --hoist",
+		"build": "lerna run build",
+		"clean": "lerna run clean",
+		"test": "lerna run test --stream",
+		"build-docs": "lerna run build-docs",
+		"link-global": "npm link ./libraries/botbuilder",
```

Comment:
Whats the need for this? Lerna should be managing all of the symlinks.

Reply:
It was to make it easier for people to link to a clone of the botbuilder-js project from outside the repo but I can remove it.